Formula/mmseqs2.rb

```diff
@@ -0,0 +1,46 @@
+class Mmseqs2 < Formula
+  desc "Software suite for very fast protein sequence search and clustering"
+  homepage "https://mmseqs.org/"
+  url "https://github.com/soedinglab/MMseqs2/archive/1-c7a89.tar.gz"
+  version "1-c7a89"
+  sha256 "e756a0e5cb3aa8e1e5a5b834a58ae955d9594be1806f0f32800427c55f3a45d5"
+
+  depends_on "cmake" => :build
+  depends_on "gcc"
+  depends_on :arch => :x86_64
+
+  fails_with :clang do
+    cause "MMseqs2 requires OpenMP support"
+  end
+
+  resource "documentation" do
+    url "https://github.com/soedinglab/MMseqs2.wiki.git", :revision => "6dbd3666edb64fc71173ee714014e88c1ebe2dfc"
+  end
+
+  def install
+    # version information is read from git, next MMseqs2 version will include a cmake flag so we do not need this hack
+    inreplace "src/version/Version.cpp", /.+/m, "const char *version = \"#{version}\";"
+
+    args = *std_cmake_args << "-DHAVE_TESTS=0" << "-DHAVE_MPI=0"
+
+    args << "-DHAVE_SSE4_1=1" if build.bottle?
+
+    system "cmake", ".", *args
+    system "make", "install"
+
+    pkgshare.install resource("documentation")
```

Comment:
probably should be `doc.install` but I can't see the contents since that URL doesn't seem to exist